NSS-Day-Cohort-66/rare-api-extra-medium
# Get All Comments Test

#Created the first test for Categories

###Steps to test

pull down Jf_api_tests in the api
run this test in your terminal
expected outcome
Found 10 test(s).
Creating test database for alias 'default'...
System check identified no issues (0 silenced).
.....
Ran 10 tests in 0.150s

OK
Destroying test database for alias 'default'...
